README.md

```diff
@@ -1,5 +1,36 @@
 # Cookiecutter Relecloud
 
+[![Github Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2Fkjaymiller%2Fcookiecutter-relecloud%2Fbadge%3Fref%3Dmain&style=flat-square)](https://actions-badge.atrox.dev/kjaymiller/cookiecutter-relecloud/goto?ref=main)
+
+## Table of Contents
+- [Cookiecutter Relecloud](#cookiecutter-relecloud)
+  - [Table of Contents](#table-of-contents)
+  - [What is Cookiecutter-Relecloud](#what-is-cookiecutter-relecloud)
+  - [Deployment Options](#deployment-options)
+  - [Deploying your cookiecutter template](#deploying-your-cookiecutter-template)
+  - [Getting Updates from this Template](#getting-updates-from-this-template)
+  - [Running your Deployment via DevContainer/Github Codespaces](#running-your-deployment-via-devcontainergithub-codespaces)
+  - [Deploy your template to Azure](#deploy-your-template-to-azure)
+
+## What is Cookiecutter-Relecloud
+
+### What is cookiecutter
+Cookiecutter is a command-line utility that creates projects from cookiecutters (project templates), e.g. creating a Python package project from a Python package project template.
+
+### What is relecloud
+Relecloud is a sample web app created by the Microsoft Python Cloud Advocacy team.
+
+### What makes Cookiecutter-Relecloud different
+
+Cookiecutter-Relecloud is a cookiecutter template that allows you to create a relecloud project with your choice of web framework, database, and deployment options.
```

Comment:
Actually, you could remove most of the headers and just have this be three paragraphs. The "what makes it different?" confuses me since I don't know what it's different from.